### DISCUSSION TITLE
Should we add “testing” label for issues?

### DISCUSSION THREAD

**Discussion by ceylingebes** [bounswe/bounswe2024group9#19] [General]
Also, should I open a new issue for testing tasks every week (like testing for week 1 etc.) or open a single issue and keep it opened for the rest of this term and link that issue for every week’s progress? What do you think?

P.S.: I could’ve ask directly on Discord but I wanted to see how Discussions work :p

**Comment by kristinatrajkovski** [comment]
Sorry for the late reply :) I personally do not think that we should add a specific label for that. This will be a part of your weekly effort, but I do not think we should have an issue for this. As the project goes along I think we will have more reviewers for each task as well. However I did add an "update" label and an "invalid" label existed before as well, so if you encounter a problem while testing you may assign an issue to whomever you need to, you can open an issue and fix the problem yourself, or in any case you may be a part of fixing the problem. I hope this helps and I am happy to discuss this more.
